src/Drivers/check_wfc.cpp

```diff
@@ -68,41 +89,51 @@ int main(int argc, char **argv)
 
   char *g_opt_arg;
   int opt;
-  while ((opt = getopt(argc, argv, "hvVs:g:i:r:f:")) != -1)
+  while(optind < argc)
```

Comment:
I feel the old code is fine. We only need to see null in the wfc_name default and the later check will catch the issue. I think all the other defaults are fine.

Reply:
this code allows us to keep a default for quick usage, and disallows something like `check_wfc J1`, which currently will check J2 instead without explanation. It also catches any other unknown arguments with a help message.

What is the disadvantage of having this?

Reply:
Now I see what you mean.